src/t.jl

```diff
@@ -105,6 +105,9 @@ values in vectors `x` and `y` come from a distribution with mean `μ0` against t
 alternative hypothesis that the distribution does not have mean `μ0`.
 
 Implements: [`pvalue`](@ref), [`confint`](@ref)
+    
+!!! note
+    This test is also known as a dependent samples t-test, dependent means t-test or correlated pairs t-test.
```

Comment:
```suggestion
    This test is also known as a dependent samples t-test.
```

Correlated pairs t-test has 732 hits against "dependent samples t-test" which has hundreds of thousands, and I don't know what a "dependent mean" could be.